src/lib.rs

```diff
@@ -58,9 +58,8 @@
 
 #[cfg(feature="simd_support")] extern crate packed_simd;
 
-extern crate rand_jitter;
-#[cfg(feature = "rand_os")]
-extern crate rand_os;
+#[cfg(feature = "getrandom")]
+extern crate getrandom;
```

Comment:
I think we can omit this in 2018 edition.

Reply:
Yes, but we haven't yet switched Rand to 2018 Rust (and don't plan to yet).

Reply:
Why not? MSRV for almost all users will require post-2018 version.